### PR TITLE
Add Hyperdrive example with Nile database

### PR DESCRIPTION
**Pull request by gwenshap**
### Summary

Adding an example of using Hyperdrive with Nile database

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/hyperdrive/examples/nile.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  order: 3
+description: Connect Hyperdrive to a Nile database instance.
+---
+
+import { Render } from "~/components";
+
+This example shows you how to connect Hyperdrive to a [Nile](https://thenile.dev) PostgreSQL database instance.
+
+Nile is PostgreSQL re-engineered for multi-tenant applications. Nile's virtual tenant databases provides you with isolation, 
+placement, insights and more for your tenant's data and embedding. You can learn more in [Nile documentation](https://www.thenile.dev/docs/getting-started/whatisnile).
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/nile.mdx]:
```suggestion
Nile is PostgreSQL re-engineered for multi-tenant applications. Nile's virtual tenant databases provide you with isolation, placement, insight, and other features for your tenant's data and embedding. Refer to [Nile documentation](https://www.thenile.dev/docs/getting-started/whatisnile) to learn more.
```

File: src/content/docs/hyperdrive/examples/nile.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## 1. Allow Hyperdrive access
+
+You can connect Cloudflare Hyperdrive to any Nile database in your workspace using its connection string - either with new set of 
+credentials or using existing set.
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/nile.mdx]:
```suggestion
You can connect Cloudflare Hyperdrive to any Nile database in your workspace using its connection string - either with a new set of credentials, or using an existing set.
```

File: src/content/docs/hyperdrive/examples/nile.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### Nile Console
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/nile.mdx]:
```suggestion
### Nile console
```

File: src/content/docs/hyperdrive/examples/nile.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+To get a connection string from Nile Console:
+
+1. Log in to [Nile Console](https://console.thenile.dev) and select a database 
+2. On the left hand menu, click on "Settings" (the bottom-most icon) and then select "Connection"
+3. Click on the elephant, and you'll see a connection string show up. 
+4. If you want a new pair of credentials for use with Cloudflare Hyperdrive (recommended!), click on "Generate credentials".
+5. Copy the connection string without the "psql" part
+
+You'll end up with a connection string similar to this:
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/nile.mdx]:
```suggestion
To get a connection string from Nile console:

1. Log in to [Nile console](https://console.thenile.dev), then select a database.
2. On the left hand menu, click **Settings** (the bottom-most icon) and then select **Connection**.
3. Click on the elephant, and you will see a connection string.
4. If you want a new pair of credentials to use with Cloudflare Hyperdrive (recommended), click **Generate credentials**.
5. Copy the connection string without the "psql" part.

You will have obtained a connection string similar to the following:
```

File: src/content/docs/hyperdrive/examples/nile.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+### Nile Console
+
+To get a connection string from Nile Console:
+
+1. Log in to [Nile Console](https://console.thenile.dev) and select a database 
+2. On the left hand menu, click on "Settings" (the bottom-most icon) and then select "Connection"
+3. Click on the elephant, and you'll see a connection string show up. 
+4. If you want a new pair of credentials for use with Cloudflare Hyperdrive (recommended!), click on "Generate credentials".
+5. Copy the connection string without the "psql" part
```
Comment (thomasgauvin) [inline src/content/docs/hyperdrive/examples/nile.mdx]:
```suggestion
3. Select the PostgreSQL logo to show the connection string. 
4. Select "Generate credentials" to generate new credentials.
5. Copy the connection string (without the "psql" part).
```

**Comment by gwenshap** [comment]
Thanks for all the feedback, guys. It is much more readable now!

**Review by thomasgauvin — APPROVED** [review]

**Comment by thomasgauvin** [comment]
> Thanks for all the feedback, guys. It is much more readable now!

Thanks for the contribution! I had started poking around with Nile postgres last week, so this is good timing!

**Comment by Oxyjun** [comment]
> Thanks for all the feedback, guys. It is much more readable now!

Thanks for the PR! We're running the pipeline now to merge :D

**Pull request merged**